### ISSUE TITLE
Add term melorheostosis

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10474]
**Preferred term label:**
Melorheostosis

**Synonyms**
-
**Definition (free text, please give PubMed ID)**
Regions of sclerosing hyperostosis in the cortex of long bones, which might have a radiologic appearance of “dripping candle wax”. PMID: 31129707

Comment: Melorheostosis is a disease (omim155950), but is also used a feature or phenotype. This is mentioned in OMIM as well: Melorheostosis may sometimes be a feature of Buschke-Ollendorff syndrome (BOS; [166700](https://www.omim.org/entry/166700)): (https://www.omim.org/entry/155950)

**Parent term (use hpo.jax.org/app)**
Abnormal long bone morphology HP:0011314

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Buschke-Ollendorf syndrome (omim166700) PMID: 26708699

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@lisannesw 
Adding new term: 
- ID: HP:6000817
- label: Melorheostosis
- parents: [Abnormal long bone morphology (HP:0011314)](https://hpo.jax.org/app/browse/term/HP:0011314)
- pmid: PMID:31129707
- def: Regions of sclerosing hyperostosis in the cortex of long bones, which might have a radiologic appearance of dripping candle wax.
- comment: The word Melorheostosis is used to denote a disease (OMIM:155950), but is also used a feature or phenotype, e.g., Buschke-Ollendorff syndrome (OMIM:166700).
- orcid: orcid:0009-0007-1020-5412
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10474
- synonyms:
